lib/api.rb

```diff
@@ -1,4 +1,6 @@
 module Api
+  VERSION = ManageIQ::Api::VERSION
+  SUPPORTED_VERSIONS = [VERSION].freeze
```

Comment:
this is fine,  essentially yes, the Gem version is the current (default) version, the above SUPPORTED_VERSIONS could be extended to include versions in addition to VERSION that the code differentiates. i.e. /api/v4.0.0/collection  vs. /api/v3.0.0/collection.   Code defined vs. config defined is better anyways.